### PR TITLE
Feature/logout

### PR DESCRIPTION
**Pull request by codeTony22**
Fixed a mistake made in a previous commit, added a log in, log out button to the nav-bar. The login button shows when the user is not authenticated, once the user is logged in the Log in nav-bar item shows.

### CODE REVIEW

File: webapp/geek_text/templates/geek_text/base.html [hunk 1]
```diff
@@ -28,6 +28,15 @@
 		    </ul>
 		    <ul class="nav navbar-nav navbar-right">
 		      <li><a href="#"><span class="glyphicon glyphicon-user"></span>Account</a></li>
+		      {% if user.is_authenticated %}
```
Comment (loksonarius) [inline webapp/geek_text/templates/geek_text/base.html]: The pattern is 👍 , the indentation is 💩

**Review by loksonarius — CHANGES_REQUESTED** [review]
Please address the poor indentation around the Login/Logout links in the nav bar.

File: webapp/geek_text/urls.py [hunk 2]
```diff
@@ -17,11 +17,12 @@
 from django.conf.urls import url, include
 from django.contrib import admin
 from django.contrib.auth import views as auth_views
+
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^login/$', auth_views.login , name='login'),
+    url(r'^logout/$', auth_views.LogoutView.as_view(template_name='browse.html') , name='logout'),
```
Comment (FreakoNico) [inline webapp/geek_text/urls.py]: I'm confused what this line does... why does the logout url use the browse.html? Is this just a redirect to the browse page after the user logs out?

Comment (codeTony22) [reply webapp/geek_text/urls.py]: That's exactly what it does. After the user is logged out that is the template that it renders.

**Pull request closed without merging**